core/positions/engine.go

```diff
@@ -314,6 +358,18 @@ func (e *Engine) Update(ctx context.Context, trade *types.Trade, passiveOrder, a
 	e.positionUpdated(ctx, buyer)
 	e.positionUpdated(ctx, seller)
 
+	var buyerSize, sellerSize uint64
+	if buyer.size > 0 {
+		buyerSize = uint64(buyer.size)
+	}
+	if seller.size > 0 {
+		sellerSize = uint64(seller.size)
+	}
+	e.partiesOpenInterest[buyer.partyID].RecordLatest(buyerSize)
+	e.partiesTradedVolume[buyer.partyID] += trade.Size
+	e.partiesOpenInterest[seller.partyID].RecordLatest(sellerSize)
+	e.partiesTradedVolume[seller.partyID] += trade.Size
```

Comment:
this naming is really not great. It's tracking the size not the tradeVolume, and what's more confusing is that later when you multiply by markPrice which actually gives you the volume, you call it... tradedVolume... 

`tradedVolume := num.UintZero().Mul(num.NewUint(oi), m.markPrice)`